app/src/main/java/org/sopt/and/extension/ModifierExtension.kt

```diff
@@ -0,0 +1,34 @@
+package org.sopt.and.extension
+
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.combinedClickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+
+@OptIn(ExperimentalFoundationApi::class)
+inline fun Modifier.noRippleClickable(
+    crossinline onClick: () -> Unit = {},
+    crossinline onLongClick: () -> Unit = {}
+): Modifier = composed {
+    this.clickable(indication = null,
+        interactionSource = remember { MutableInteractionSource() }) {
+    }
+    this.combinedClickable(
+        onClick = { onClick() },
+        onLongClick = { onLongClick() },
```

Comment:
여기엔 중괄호를 안써도 될 것 같아요!
인자가 없거나, 람다와 함수의 인자 순서가 같다면 중괄호를 사용하지 않아도 된답니다!
```suggestion
        onClick = onClick,
        onLongClick = onLongClick,
```

Reply:
가독성을 위해 좋을 것 같네요 감사합니다!